label_studio/users/models.py

```diff
@@ -41,6 +46,15 @@ def _create_user(self, email, password, **extra_fields):
 
         return user
 
+    def get_queryset(self):
+        qs = super().get_queryset()
```

Comment:
I debated putting this behind the feature flag as well but as it modifies the manager which required changes to the migrations themselves I'm leaning towards leaving it without the flag - this wont have an effect until users start being soft deleted.

Reply:
That was my inclination as well, the part that will naturally make this actionable will be feature flagged and until actually utilized will have no effect. The other argument would be if something were to break as a result of this change, but again this is under test so that should not be an issue.